gadget/validate.go

```diff
@@ -173,46 +183,47 @@ func validateReservedLabels(vs *VolumeStructure, reservedLabels []string) error
 	return nil
 }
 
-func ensureRolesConsistency(roles map[string]*roleInstance, expectedSeed bool) error {
+func ensureRolesConsistency(roles map[string]*roleInstance, seedExpected bool) error {
```

Comment:
was it intentional to swap this around? reducing the diff even if maybe it sounds better would still seem a win

Reply:
Yes, `seedExpected` sounded more suggesting a boolean (maybe `seedIsExpected` would be even better) than `expectedSeed` to me that sounded like a name for the seed or something like that. I don't mind reverting but tbh the diff is already quite big and changing this one won't make such a difference imo.